jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/src/test/java/org/eclipse/jetty/ee10/osgi/test/TestOSGiUtil.java

```diff
@@ -13,15 +13,18 @@
 
 package org.eclipse.jetty.ee10.osgi.test;
 
-import java.io.File;
+import java.net.URI;
```

Comment:
There are several changes in this file that are extraneous to the deployment manager, which makes reviewing harder. it would have been better to raise a separate issue for improvements.

Reply:
I'll make these a different PR

Reply:
Moved the non-deploy related changes to OSGi to a new PR #12845